openreview/tools.py

```diff
@@ -577,11 +577,14 @@ def subdomains(domain):
         full_domain = domain
     domain_components = [c for c in full_domain.split('.') if c and not c.isspace()]
     domains = ['.'.join(domain_components[index:len(domain_components)]) for index, path in enumerate(domain_components)]
-    valid_domains = [d for d in domains if not tld.is_tld(d)]
-
-    for d in valid_domains:
-        if d in duplicate_domains and duplicate_domains[d] not in valid_domains:
-            valid_domains.append(duplicate_domains[d])
+    valid_domains = []
```

Comment:
I was thinking something simpler:

```
unique_valid_domains = [duplicate_domains.get(d, d) for d in valid_domains]

return unique_valid_domains
```

valid_domains is still the the result of checking the domains against tld.